react-project-aston/src/store/auth-context.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+
+const AuthContext = React.createContext({
+  isLoggedIn: false,
+  login: (user) => {},
+  logout: () => {},
+  user: null,
+});
+
+export const AuthProvider = (props) => {
+  let initialUser;
+  const savedUser = localStorage.getItem('user');
+
+  if (savedUser) {
+    initialUser = savedUser;
+  }
+
+  const [user, setUser] = useState(initialUser);
```

Comment:
Выглядит так, что это все можно сократить.

```suggestion
  const [user, setUser] = useState(() => localStorage.getItem('user'));
```